### ISSUE TITLE
Actions and Keybinding lists are not iterable

### ISSUE THREAD

**Issue by engbergandreas** [OpenSpace/OpenSpace#3419]
The lists of Actions and Keybindings in Profile Editor are not iterable beyond the second item using arrow keys. The problem is as soon as you go down the list focus is shifted to modifying the properties of said action or keybinding. Focus should probably either be kept on the list or have some edit button that the user can tab to, to edit the property (or use spacebar to select).

"bar" and "Home" are not accessible in the below screenshot using keyboard
![image](https://github.com/user-attachments/assets/dba37b22-9b3b-4bd3-a165-0ae9ebd35b30)
